### ISSUE TITLE
Reset admin settings to default does not work

### ISSUE THREAD

**Issue by schuco** [Neriderc/GVExport#364]
Method getAdmin?reset=1Action() not found in _GVExport_

**Comment by Neriderc** [comment]
Nooo! I thought we were all set!

I don't think this has been touched recently. When's the last time this worked for you?

Can you post the URL? I'm mostly interested in the bit after the domain name. I know your URLs are structured differently to mine, it would be good to see what that looks like as I suspect that's where the issue lies.

**Comment by schuco** [comment]
https://brandneu.schulte-coerne.de/index.php?route=%2Fmodule%2F_GVExport_%2FAdmin?reset=1

I never tried that before, so I don't know how long the error exist.

I tried it because you mentioned older saved settings would not work with 2.1.17. But it seems they do work in my case

**Comment by Neriderc** [comment]
Ah ok I see a problem there. I'll look at fixing this when I get a chance (I'm not at home and won't be until tonight - but should be easy to fix (it's again related to pretty URLs vs standard URLs)).

> I tried it because you mentioned older saved settings would not work with 2.1.17. But it seems they do work in my case

That's for people upgrading from 2.1.16 (the previous release). You've been installing pre-release versions so have effectively already been running 2.1.17 for a while. The only changes made since you last updated were probably just some translation changes and wording changes, a tiny bug that no one had mentioned but I thought of in the shower, and changing the version number in the code so webtrees prompts for the update.

**Comment by hartenthaler** [comment]
I can confirm that bug. It is only in my test system, that is not using pretty URLs (and still uses http).

**Comment by Neriderc** [comment]
I've just pushed a change to the main branch. Could someone check that this works now with the standard URLs?

**Comment by schuco** [comment]
I just tried th new main:
As soon as I click the admin settings to open:
![grafik](https://user-images.githubusercontent.com/5896925/220067477-d56b9d0a-06c1-4d62-adaa-4b494f94113c.png)

**Comment by Neriderc** [comment]
Thanks. What does the URL look like?

**Comment by Neriderc** [comment]
Never mind, I think I found the issue. Can you try again?

**Comment by schuco** [comment]
still:
![grafik](https://user-images.githubusercontent.com/5896925/220072604-6bc5e834-8d70-481a-86eb-e00f02d1d833.png)
URL:
https://brandneu.schulte-coerne.de/index.php?route=%2Fmodule%2F_GVExport_%2FAdmin

**Comment by schuco** [comment]
@Neriderc  @hartenthaler I will leave for vacation next two weeks and will only have my Android tablet ond phone. How do I load a main with Android browser? I dont find a green "Code" tab to download a zip like in Windows

**Comment by hartenthaler** [comment]
For me it is working now perfect.

**Comment by hartenthaler** [comment]
@schuco I'm not familiar with Android. On iPhone there is no option to download the main branch and upload it afterwards. But this is normal for the closed iPhone Apple world.

**Comment by Neriderc** [comment]
> @Neriderc @hartenthaler I will leave for vacation next two weeks and will only have my Android tablet ond phone. How do I load a main with Android browser? I dont find a green "Code" tab to download a zip like in Windows

I had a play on my android phone but could not work it out when in mobile mode. However, in Firefox for Android (and many other mobile browsers) you can click the ... and choose to enable the "Desktop site". This loads a desktop view and for me I can see the normal green Code button for downloading the main branch.

Could you try it again when you get the chance? I've fixed something that has caused issues with PHP-FPM in the past. I know it causes issues but it's hard to break the habit of using the syntax! Sorry! Hopefully this is fixed now (main branch).

**Comment by schuco** [comment]
I tried to do it at Düsseldorf airport on my Android tablet. Managed to download main in desktop mode. Hopfully I got it right to transfer to server.
But then:
Fatal error: Cannot declare class vendor\WebtreesModules\gvexport\functionsClippingsCart, because the name is already in use in /var/www/vhosts/schulte-coerne.de/brandneu.schulte-coerne.de/modules_v4/GVExport/app/functionsClippingsCart.php on line 60
